src/docs/install.md

```diff
@@ -17,47 +17,42 @@ After you have pants installed, you'll need to
 Recommended Installation
 ------------------------
 
-To set up pants in your repo, we recommend installing our self-contained `pants` bash script
-in the root (ie, "buildroot") of your repo:
+To set up Pants in your repo, we recommend installing our self-contained `pants` bash script
+in the root (i.e. the "buildroot") of your repo:
 
     :::bash
-    curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants && touch pants.ini
+    curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants
 
-The first time you run the new `./pants` script it will install the latest version of pants (using
-virtualenv) and then run it.  It's recommended though, that you pin the version of pants.  To do
-this, first find out the version of pants you just installed:
+Start by running the below command to auto-generate a `pants.ini` config file with sensible defaults.
 
     :::bash
-    ./pants -V
-    1.0.0
+    ./pants generate-pants-ini
 
-Then add an entry like so to `pants.ini` with that version:
+Running `./pants` for the first time will install the latest version of Pants, using virtualenv and first trying to use Python 3.6 and then falling back to Python 2.7. (Note that Pants 1.16.0 will be the last version to support Python 2.7.)
```

Comment:
> (Note that Pants 1.16.0 will be the last version to support Python 2.7.)

Might be worth noting this in line 8?

Reply:
I originally had this deprecation note on line 8, but it made the section seem cluttered / overwhelming and unnecessary as the 3rd line you read, so I removed it and moved it here.

I'm not sure where this line best belongs. Thoughts from others?

Reply:
Delete the sentence outright? It seems like tmi for the topic at hand. I don't really care how the install works - I just want to know requirements and what I do.